app/server/logging/jaeger/src/test/java/io/syndesis/server/logging/jaeger/service/JaegerActivityTrackingServiceTest.java

```diff
@@ -52,8 +63,9 @@ public void shouldRetainLastRetainActivityLogs() throws IOException {
         List<Activity> activities = service.getActivities("test", null, null);
         assertThat(activities).isNotNull();
 
-
         String activitiesJson = JsonUtils.writer().withDefaultPrettyPrinter().writeValueAsString(activities).trim();
+        // print the activitiesJson to replace the content of expected-activities.json
+        // System.out.println(activitiesJson);
         String expectedActivitiesJson = resource("expected-activities.json").trim();
         assertThat(activitiesJson).isEqualTo(expectedActivitiesJson);
```

Comment:
// nit
I'd use JSONAssert here instead not to rely on formatting changes or property order in the JSON.

Reply:
Done. Thanks.